modules/core/src/main/scala/com/azavea/stac4s/extensions/layer/LayerProperties.scala

```diff
@@ -0,0 +1,22 @@
+package com.azavea.stac4s.extensions.layer
+
+import cats.Eq
+import io.circe.{Decoder, Encoder}
+import io.circe.syntax._
+
+case class LayerProperties(layers: List[String])
```

Comment:
The naming convention for extension fields is `extensionName:fieldName` -- so this should be something like `layers:layers` (or `layers:memberOf` or something) (https://github.com/radiantearth/stac-spec/tree/master/extensions#prefixes)

Reply:
Thanks!

Reply:
@jisantuc renamed it to `layer:ids` (mb this would be good enough for the first pass? I expect that there would be some discussion later to talk about the field name :D)

Reply:
Yeah that's fine for now -- `layers:layers` was gonna look pretty goofy but this is fine